doc/nrf/device_guides/working_with_nrf/nrf53/qspi_xip_guide.rst

```diff
@@ -0,0 +1,321 @@
+.. _qspi_xip:
+
+Configuring external XIP
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This section describes the external XIP support for the nRF5340 SoC.
```

Comment:
```suggestion
This section describes the external execute in place (XIP) support for the nRF5340 SoC.
```
Maybe not needed, but I have seen it done so in other nRF53 guides: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/device_guides/working_with_nrf/nrf53/nrf5340.html#execute-in-place-xip-configuration

Reply:
Added:)